openapi/platform/platform.yaml

```diff
@@ -399,32 +399,6 @@ paths:
         [Status](https://vendasta.stoplight.io/docs/openapi-specs/docs/1.-Overview/Versioning.md): `Proposed`
 
         Removes the specified location(s) from the contact. Locations that have already been removed will be ignored.
-      requestBody:
```

Comment:
I removed this body from the proposal as the delete verb wouldn't have a request payload.